src/components/page-layout/components/example-index.js

```diff
@@ -158,6 +166,16 @@ export default class ExampleIndex extends React.PureComponent {
       <div className="mb18 mb0-mxl">
         <AsideHeading>Filters</AsideHeading>
         <div className="grid grid--gut6">
+          {availableFilters.search && (
+            <FilterSection
+              title="Search"
+              id="search"
+              activeItem={search}
+              handleInput={this.handleInput}
+              isText={true}
+              placeholder="Search title and description&hellip;"
```

Comment:
Maybe "Search for an example..."?

Reply:
Not sure we want to use "example" as filters can show up on tutorials and troubleshooting pages too 🤔

Reply:
Let's leave for now and update later if we find an alt!